requirements.txt

```diff
@@ -1,15 +1,16 @@
-h5py>=2.10.0
-matplotlib>=1.4.3
-numpy>=1.12.1
-pandas>=0.17.0
-scipy>=0.15.1
-simplejson>=3.10.0
-pillow
-argschema
+argschema<2.0.0
 allensdk
 dictdiffer
-pyabf<2.3.0
-pynwb>=1.3.2,<2.0.0
```

Comment:
Why is the minimum pynwb reverting from 1.3.2 back to 1.3.1?  I thought we were attempting to match [AllenSDK](https://github.com/AllenInstitute/AllenSDK/blob/master/requirements.txt#L22)